src/Command/Mute.js

```diff
@@ -0,0 +1,117 @@
+const Command = require('../Command');
+
+module.exports = Command.extend({
+    commandName: 'mute',
+    commandAliases: ['ban'],
+    advertisable: false,
+    processMessage: function (message, tokens) {
+        let hasRoles = message.member !== null && typeof message.member.roles !== 'undefined';
```

Comment:
You can safely move this check into `this.memberIsAdministrator` - which can return false under these conditions